webapp/src/components/link_tooltip/tooltip.css

```diff
@@ -0,0 +1,116 @@
+:root {
+    --light-gray: #6a737d;
+    --gray: #586069;
+    --dark-gray: #24292e;
+    --blue: #274466;
+    --light-blue: #eff7ff;
+}
+
+@media (min-width: 544px) {
+    .github-tooltip--large {
+        min-width: 320px;
+    }
+}
+
+/* Github Tooltip */
+.github-tooltip .box {
+    background-color: white;
+    box-shadow: 0 1px 15px rgba(27,31,35,.15);
+    position: relative;
+    width: 360px;
+}
+
+.github-tooltip--bottom-left, .github-tooltip--top-left {
+    left: -9px;
+    margin-left: 0;
+}
+
+/* Header */
+.github-tooltip .header {
+    color: var(--light-gray);
+    font-size: 12px;
+}
+
+.github-tooltip .header a {
+    text-decoration: none;
+    color: var(--gray);
+    display: inline-block;
+}
+
+/* Body */
+.github-tooltip .body > span {
+    line-height: 1.25;
+}
+
+/* Info */
+.github-tooltip .tooltip-info {
+    line-height: 1.25;
+    display: flex;
+    flex-direction: column;
+}
+.github-tooltip .tooltip-info > a, .github-tooltip .tooltip-info > a:hover {
+    display: block;
+    text-decoration: none;
+    color: var(--dark-gray);
+}
+.github-tooltip .tooltip-info > a span {
+    color: var(--light-gray);
+}
+
+.github-tooltip .tooltip-info h5 {
+    font-weight: 600;
+    font-size: 14px;
+    display: inline;
+}
+
+.github-tooltip .tooltip-info .markdown-text {
+    -ms-overflow-style: none;
+    -webkit-hyphens: auto;
+    -ms-hyphens: auto;
+    hyphens: auto;
+    max-height: 150px;
```

Comment:
Well... actually, I gave this a though and decided to do it like that. The thing is, the scrolling is enabled, so you should be able to scroll down there but I have hidden the scrollbar because I thought it was more "elegant" not to show it, but it might seem a bit weird.
Either I could show the scrollbar or I could play around with the 'react-markdown' so it only renders an amount of HTML tags... There are several options in which this could be done... I could also just convert the whole text to one only <p> with no format and add ellipsis on overflow, i don't know.... 
I would say that, because of the scope of this issue (which was not actually including the pr/issue description on the tooltip), I would rather decide between keeping it like this or un-hiding the scrollbar. Thoughts?

Reply:
I'm wondering how much a scrollbar would be actually used. The tooltip is mostly "for a quick glance" and if a user wants for information, the link to the issue/pr is right there. 


@asaadmahmood I would like to hear your toughs on this. Is the current solution acceptable to you?

Reply:
I feel it could be useful... Even though it's a quick glance, sometimes we get a bit lazy about having to open the link, wait the page to load, etc etc. Imagine that maybe you know there's a useful link inside the description that you know is placed at the bottom of it... just scrolling will help you out... But yeah, up to others opinions :)

Reply:
@hanzei Sorry for replying late guys. I don't think a scrollbar should be needed here, we can just have a max-height or max number or elements, without it cutting off content.

I would also be willing to add a "See more" link above the labels which would just open the PR (similar to the title link) to be very obvious.

Reply:
it currently has a max-height, which cuts the text off and that's why I've added the ability to scroll (just that I've hidden the scrollbar, but you are allowed to scroll) @asaadmahmood I'm not so sure which is the solution you propose.
Regarding the "See more" link, no problem!